.devcontainer/Dockerfile

```diff
@@ -1,5 +1,8 @@
 FROM mcr.microsoft.com/devcontainers/universal:2-linux
 
+WORKDIR /data
```

Comment:
Add the data files to `/data`; they'll show up in the devcontainer automatically.

Reply:
its already in `.devcontainer/data/`

Reply:
is the copy redundant?

Reply:
I mean move the data to the root of the repo